pkg/util/engine/engine.go

```diff
@@ -39,3 +39,17 @@ func IsTiFlashHTTPResp(store *pdhttp.MetaStore) bool {
 	}
 	return false
 }
+
+// IsTiFlashWriteHTTPResp tests whether the store is either
+// - a TiFlash node under Classic kernel
+// - a TiFlash write node under NextGen kernel
+// from a PD HTTP response.
+// This function will return false for TiFlash compute node under NextGen kernel.
+func IsTiFlashWriteHTTPResp(store *pdhttp.MetaStore) bool {
+	for _, label := range store.Labels {
+		if label.Key == "engine" && label.Value == "tiflash" {
```

Comment:
for classic, it's "tiflash"? can you put this info about those string literal on the comment

Reply:
I've added comments about the label of tiflash/tiflash-write/tiflash-compute on this file.

https://github.com/pingcap/tidb/pull/64709/files#diff-103eeb6beca237f72e79f8453aa2303b502201079221ef53229424f1f11f5911R22-R25